src/databricks/labs/remorph/resources/reconcile/dashboards/failed_table_list.sql

```diff
@@ -0,0 +1,26 @@
+select
+main.recon_id,
+main.source_type,
+main.report_type,
+main.source_table.`catalog` as source_catalog,
+main.source_table.`schema` as source_schema,
+main.source_table.table_name as source_table_name,
+CASE
```

Comment:
Please update the `CASE..WHEN..END` statement to `if(cond, exp1, exp2)`... to simplify the statement and improve readability.

```suggestion
IF(ISNULL(MAIN.SOURCE_TABLE.CATALOG)
    , CONCAT_WS('.', MAIN.SOURCE_TABLE.SCHEMA, MAIN.SOURCE_TABLE.TABLE_NAME)
    , CONCAT_WS('.', MAIN.SOURCE_TABLE.CATALOG, MAIN.SOURCE_TABLE.SCHEMA, MAIN.SOURCE_TABLE.TABLE_NAME)) AS source_table,
```

Reply:
New files were added